akka-osgi/build.sbt

```diff
@@ -1,6 +1,7 @@
 import akka.{ AkkaBuild, Dependencies, Formatting, OSGi }
 
 AkkaBuild.defaultSettings
+AkkaBuild.dontPublishSettings
```

Comment:
This boils down to the question if everything that has `publishArtifact := false` should also be excluded from the dist zip.

If yes then we could get rid of `Dist.distExclude` altogether and just filter on `publishArtifact` there??